README.md

```diff
@@ -17,11 +17,9 @@ Developed by [CatalystNeuro](http://catalystneuro.com/).
 ## Usage:
 ### Supported file types:
 #### Imaging
-1. HDF5
-2. TIFF
-3. STK
-4. FLI
```

Comment:
I am not sure which formats are FLI and STK. Maybe removing this is wrong but I could not find them in the library.

Reply:
@h-mayorquin These can be found implemented here: https://github.com/catalystneuro/roiextractors-gpl/blob/master/roiextractors_gpl/extractors/bioformatsimagingextractors/bioformatsimagingextractors.py

You are free to move them over to this project, but the dependencies *must* be imported lazily due to the licensing difference (never put GPL license in our requirements/setup). It's a secondary software that the user must have installed separately.

We'll see if we can dig up the sample data they were based on but given it's been 2+ years that may be difficult.